### PR TITLE
fixed bug with rstrip

### PR DESCRIPTION
**Pull request by adnanhemani**
[No] Wrote test for feature
[X] Added changes in the Changelog section in README.md
[X] Bumped version number (delete if unneeded)

**Changes proposed:**
Fixed the bug as described in #338. Was caused as a side-effect of using rstrip which will truncate a set of characters from the back of a string rather than a substring - causing a label like "proportion" to get truncated to "proporti" because "i" was the first letter not in the set of characters " ,c,o,u,n,t".

### CODE REVIEW

**Comment by adnanhemani** [comment]
Again, I'm unsure as to how to test this properly.

File: Untitled.ipynb [hunk 1]
```diff
@@ -0,0 +1,116 @@
+{
```
Comment (davidwagner) [inline Untitled.ipynb]: Let's not push a notebook called Untitled.ipynb to the top-level directory.  If this has some tests, let's put it under tests/ with a suitable name.

File: datascience/tables.py [hunk 2]
```diff
@@ -2586,7 +2586,12 @@ def prepare_hist_with_bin_column(bin_column):
             # This code is factored as a function for clarity only.
             weight_columns = [c for c in self.labels if c != bin_column]
             bin_values = self.column(bin_column)
-            values_dict = [(w.rstrip(' count'), (bin_values, self.column(w))) for w in weight_columns]
+            print(bin_column)
```
Comment (davidwagner) [inline datascience/tables.py]: We shouldn't print things to stdout.  Perhaps the print statements are a remnant of debugging, and need to be deleted?

File: datascience/tables.py [hunk 3]
```diff
@@ -2662,6 +2667,8 @@ def draw_hist(values_dict):
                     x_unit = ' (' + unit + ')' if unit else ''
                     if len(weights) == n:
                         vargs['weights'] = weights[i]
+                    # print(hist_name)
```
Comment (davidwagner) [inline datascience/tables.py]: Presumably these can be deleted.

**Comment by davidwagner** [comment]
> I'm unsure as to how to test this properly.

If it doesn't mess up any of the histograms in tests/Charts.ipynb, and fixes the problem in #337, I'd say that we have probably tested it sufficiently.

In the long run, we might be able to automate some of the testing for histograms using https://matplotlib.org/devel/testing.html#writing-an-image-comparison-test.

File: Untitled.ipynb (hunk 1 above)
Comment (adnanhemani) [reply Untitled.ipynb]: Whoops! Thanks for catching!

**Comment by adnanhemani** [comment]
Made all the fixes - thanks for the assist!

**Pull request merged**